### ISSUE TITLE
3.2.0 Won't function on Windows < Win 10

### ISSUE THREAD

**Issue by narcolepticinsomniac** [persepolisdm/persepolis#689]
Win 8.1
Persepolis 3.2.0

After installing exe, Persepolis won't open:

![2019-09-17 05_04_32-Error](https://user-images.githubusercontent.com/14100003/65028366-f3385880-d909-11e9-974d-bc445ac11ac8.png)

**Comment by alireza-amirsamimi** [comment]
Hello

a bug in [new version of QT](https://stackoverflow.com/questions/48514027/spyder3-python-ide-does-not-start-this-windows-version-does-not-support-the-re/51549759) caused this error on old version of MS Windows.

We must change the codes for solving this problem.

Thanks for reporting

**Comment by narcolepticinsomniac** [comment]
Can't wait to try it out. I've been watching releases to see if you guys would fix the minimize to system tray on exit issue. The other improvements sound promising as well. Hope you can fix it soon, and keep up the good work!

**Comment by mostafaasadi** [comment]
Hello
Please test this package and tell me the results :)
https://mega.nz/#!664EDYaR!6e3qn8u-S967TJ_dKPFTj36N6xTYlBoCcoEtYm5Rqgc

**Comment by narcolepticinsomniac** [comment]
Nope, unfortunately, same stupid Bluetooth API error. Persepolis won't open.

**Comment by narcolepticinsomniac** [comment]
Btw, I was curious so I tested that mega exe on a different Win 7 machine, and Persepolis seems to install and launch fine. Can't get the same installer to work on Win 8.1 though. I'm uninstalling real thoroughly before re-installing, so I'm not sure how the previous borked installer could affect the updated one. It's either something like that, or the fix works for Win 7 but not Win 8 though.

**Comment by narcolepticinsomniac** [comment]
Actually, judging by the first comment in the [related issue](https://stackoverflow.com/questions/48514027/spyder3-python-ide-does-not-start-this-windows-version-does-not-support-the-re/51549759) you linked, it seems like this might be strictly a Win 8.1 problem to begin with. Still hoping for a fix. I was looking forward to testing the new version.

**Comment by narcolepticinsomniac** [comment]
Been a month, and Persepolis still won't open after installing on Win 8.1. I've been looking for a download manager to replace uGet for a while now.

**Comment by narcolepticinsomniac** [comment]
Come on guys. It's almost 4 months later, and Persepolis still can't be installed on Win 8.1. Isn't this as simple as either [specifying the import](https://stackoverflow.com/questions/55205708/why-pyqt5-import-show-a-error-about-windows-and-bluetooth) or reverting QT to a version which isn't bugged?

**Comment by tkm22** [comment]
Same issue. Could you please fix it, thanks.
